src/components/Navbar.jsx

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import { NavLink } from 'react-router-dom';
 
 const Navbar = () => (
-  <nav className="navbar">
-    <div className="navbar-logo">
-      <h1>Bookstore CMS</h1>
+  <nav className="flex px-14 gap-2 bg-white shadow-md w-full h-20">
+    <div className="flex align-middle h-20">
+      <h1 className="m-auto w-60 h-9 font-bold text-3xl text-center text-sky-600">Bookstore CMS</h1>
     </div>
-    <div className="navbar-links">
-      <ul>
+    <div className="w-60 mt-6 mr-0 mb-0 mx-0">
+      <ul className="flex justify-center gap-2 mt-1">
         <li className="nav-link">
-          <NavLink className="nav-item" to="/books">BOOKS</NavLink>
+          <NavLink className=" text-sm text-black hover:text-gray-600 active:bg-gray-200" to="/books">BOOKS</NavLink>
         </li>
         <li className="nav-link">
-          <NavLink className="nav-item" to="/categories">CATEGORIES</NavLink>
+          <NavLink className=" text-sm  text-black hover:text-gray-600 active:bg-gray-200" to="/categories">CATEGORIES</NavLink>
         </li>
       </ul>
```

Comment:
- [ ] Please add the user icon in the header to match with the Zepline design.

### Your design

![image](https://github.com/lincoln1883/anancy-bookstore/assets/83803414/14bfa39e-be09-4114-927b-bfb19b816ae6)

### Zepline design

![image](https://github.com/lincoln1883/anancy-bookstore/assets/83803414/3f5fd7a9-d0e6-4a0d-8bb3-a3ea8e084cef)

Reply:
Done thank you.